src/bin/path_filestat.rs

```diff
@@ -129,6 +129,30 @@ unsafe fn test_path_filestat(dir_fd: wasi_unstable::Fd) {
     assert_eq!(stat.st_mtim, new_mtim, "mtim should not change");
     assert_eq!(stat.st_atim, old_atim, "atim should not change");
 
+    let new_atim = old_atim - 100;
```

Comment:
This looks unnecessary.

Reply:
Is it not though? How else are we to differentiate between a call which touched the path while reporting an `EINVAL`, from that which didn't? Or am I missing something here? :-)

Reply:
Misread atim for mtim. My bad!

Reply:
No probs! :-)